src/pageObject.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return `Conduit logo`;
+  }
 };
 
 class PageObject {
+  constructor(url, header) {
```

Comment:
you do not need to pass the header to the constructor parameters, since you will not assign it
constructor contains `header` property which is an instance of `Header` class

Reply:
I don't quite understand why the Header in the constructor is not needed here

Reply:
because the 'header' parameter is not used, so you need to remove not used parameters